src/typescript/frontend/src/components/pages/emojicoin/components/trade-emojicoin/LiquidityButton.tsx

```diff
@@ -14,7 +15,9 @@ export const LiquidityButton = (props: GridProps) => {
       {!isInBondingCurve(props.data) ? (
         <StyledContentHeader>
           <Flex width="100%" justifyContent="center">
-            <Button scale="lg">{t("Provide liquidity")}</Button>
+            <Link href={`/pools?pool=${props.data.emojis.reduce((p, c) => `${p}${c.emoji}`, "")}`}>
```

Comment:
```suggestion
            <Link href={{
              pathname: ROUTES.pools,
              query: { pool: props.data.emojis.join("") },
            }}>
```

BTW yesterday I realized we can just use the default export `import Link from "next/link";`to have access to a better Link component with a clearer prop for pathname and query.

The original one you're using wrapped it with theme and styling and wasn't made when we were using `nextjs`, so you can use it here since there's no styling here (although you could add the scramble I suppose)

It's not really a big deal whether or not we use it here so I'll approve, just in case you feel like changing it 😁

Reply:
My bad. I meant to import the `nextjs` one. Auto-import messed this one up :sweat_smile:

Reply:
Hahaha it's ok, we have too many old components lying around. I do it all the time too